src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -18,13 +19,15 @@
 
 // Use english locale, of not set, the test instance should be set to english language
 public class Steps extends ReferenceApplicationTestBase {
+	
 	protected TestProperties testProperties = TestProperties.instance();
 	protected LoginPage loginPage;
 	protected FindPatientPage findPatientPage;
 	protected String firstPatientIdentifier;
 	protected ClinicianFacingPatientDashboardPage dashboardPage;
 	protected By patientHeaderId = By.cssSelector("div.identifiers span");
 	protected InitialSetupPage initialSetupPage;
+	protected PatientVisitsDashboardPage visitsDashboardPage;
```

Comment:
Added `PatientVisitsDashboardPage` to be at global variable that can be re-usable by child classes that extends `Steps`.

Reply:
Where are you calling visitDashboardPage variable ?

Reply:
> Where are you calling visitDashboardPage variable ?

In the child class called `ClinicalVisitSteps`. See some of its calls [here](https://github.com/kdaud/openmrs-contrib-qaframework/blob/0f266d41f63d509ce8ff5a0223b58af6ac952b86/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java#L79) . However, other child classes will need this variable so declaring it at a global level reduces `code redundancy` rather than declaring this variable in every child class!